recipes/common/hyprland-modules.yml

```diff
@@ -4,7 +4,9 @@ modules:
         - hyprland
         - waybar
         - xdg-desktop-portal-hyprland
-        - brightnessctl
```

Comment:
brightnessctl is used by the wayblue default waybar config, was this deliberately removed?

Reply:
Yes, I removed it because it is already part of the common-modules:
https://github.com/wayblueorg/wayblue/blob/991a451dd67913917d4e27b4bdc888d449497464/recipes/common/common-modules.yml#L91

Reply:
got it thanks, merged